cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js

```diff
@@ -0,0 +1,112 @@
+import { TestTypes, DevTeams, Permissions } from '../../support/dictionary';
+import { ITEM_STATUS_NAMES, REQUEST_TYPES } from '../../support/constants';
+import UserEdit from '../../support/fragments/users/userEdit';
+import TopMenu from '../../support/fragments/topMenu';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import Requests from '../../support/fragments/requests/requests';
+import Users from '../../support/fragments/users/users';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import NewRequest from '../../support/fragments/requests/newRequest';
+import OtherSettings from '../../support/fragments/settings/circulation/otherSettings';
+import CheckOutActions from '../../support/fragments/check-out-actions/check-out-actions';
+import RequestsSearchResultsPane from '../../support/fragments/requests/requestsSearchResultsPane';
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+
+describe('Circulation log', () => {
+  let userData = {};
+  let requestId;
+  let itemBarcode;
+  const testData = {
+    folioInstances: InventoryInstances.generateFolioInstances({ count: 1 }),
+    servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+  };
+
+  before('Preconditions:', () => {
+    cy.getAdminToken();
+    ServicePoints.createViaApi(testData.servicePoint);
+    testData.defaultLocation = Location.getDefaultLocation(testData.servicePoint.id);
+    Location.createViaApi(testData.defaultLocation).then((location) => {
+      InventoryInstances.createFolioInstancesViaApi({
+        folioInstances: testData.folioInstances,
+        location,
+      });
+    });
+    OtherSettings.setOtherSettingsViaApi({ titleLevelRequestsFeatureEnabled: true });
+    cy.createTempUser([
+      Permissions.uiRequestsCreate.gui,
+      Permissions.uiRequestsView.gui,
+      Permissions.uiRequestsEdit.gui,
+      Permissions.requestsAll.gui,
```

Comment:
Please remove 'Permissions.requestsAll.gui,' permission. It's not mentioned in testcase.

Reply:
done